packages/expo-cli/src/exp.ts

```diff
@@ -321,6 +331,58 @@ Command.prototype.asyncAction = function (asyncFn: Action, skipUpdateCheck: bool
   });
 };
 
+function getStringBetweenParens(value: string): string {
+  const regExp = /\(([^)]+)\)/;
+  const matches = regExp.exec(value);
+  if (matches && matches?.length > 1) {
+    return matches[1];
+  }
+  return value;
+}
+
+function focusLastPathComponent(value: string): string {
```

Comment:
<img width="1218" alt="93028954-d2d3d480-f617-11ea-822c-6ad25ce10493" src="https://user-images.githubusercontent.com/497214/93093325-38bc6c80-f6a9-11ea-8d5d-f412b6ac56da.png">

Maybe the beginning of the path should be dimmed for lines that don't have a function name too? It seems that in this screenshot they're not?

Reply:
Here's what that looks like:
<img width="1218" alt="Screen Shot 2020-09-14 at 6 58 18 PM" src="https://user-images.githubusercontent.com/9664363/93115540-47f8e580-f6bc-11ea-8e6e-144f209aa7a7.png">

I think it's a little more jarring because the solid stack trace now has a gap in it.

Reply:
Can we put something there to fill the gap? E.g. Chrome Dev Tools shows `(anonymous function)` instead of the name, if there's no function name for the stack frame, I think.